src/legacy/server/kbn_server.d.ts

```diff
@@ -129,7 +129,7 @@ export interface KibanaCore {
     plugins: PluginsSetup;
```

Comment:
PluginsSetup contains plugin required types. they should be optional, as a plugin can be disabled. won't fixed in this PR, as required additional code changes